File: app/src/main/java/com/BlizzardArmory/diablo/account/JewelerSeason.java
package com.BlizzardArmory.diablo.account;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import org.apache.commons.lang3.builder.ToStringBuilder;

public class JewelerSeason {

    @SerializedName("slug")
    @Expose
    private String slug;
    @SerializedName("level")
    @Expose
    private Integer level;

    public String getSlug() {
        return slug;
    }

    public void setSlug(String slug) {
        this.slug = slug;
    }

    public Integer getLevel() {
        return level;
    }

    public void setLevel(Integer level) {
        this.level = level;
    }

    @Override
    public String toString() {
        return new ToStringBuilder(this).append("slug", slug).append("level", level).toString();
    }

}

File: app/src/main/java/com/BlizzardArmory/model/diablo/account/JewelerSeason.kt
package com.BlizzardArmory.model.diablo.account

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Jeweler season.
 */
@Keep
data class JewelerSeason(

    @SerializedName("slug")
    var slug: String,


    @SerializedName("level")
    var level: Int

)